packages/integration_test/lib/integration_test.dart

```diff
@@ -167,11 +167,33 @@ https://flutter.dev/docs/testing/integration-tests#testing-on-firebase-test-lab
   /// side.
   final CallbackManager callbackManager = driver_actions.callbackManager;
 
-  /// Taking a screenshot.
+  /// Takes a screenshot.
   ///
-  /// Called by test methods. Implementation differs for each platform.
-  Future<void> takeScreenshot(String screenshotName) async {
-    await callbackManager.takeScreenshot(screenshotName);
+  /// On Android, you need to call `convertFlutterSurfaceToImage()`, and set the
+  /// `framePolicy` to `LiveTestWidgetsFlutterBindingFramePolicy.fullyLive`
+  /// prior to taking a screenshot.
+  Future<List<int>> takeScreenshot(String screenshotName) async {
+    if (Platform.isAndroid) {
+      assert(
+        framePolicy == LiveTestWidgetsFlutterBindingFramePolicy.fullyLive,
+        'To take a screenshot on Android, you must set binding.framePolicy to LiveTestWidgetsFlutterBindingFramePolicy.fullyLive.',
+      );
+    }
+    reportData ??= <String, dynamic>{};
+    reportData!['screenshots'] ??= <dynamic>[];
+    final Map<String, dynamic> data = await callbackManager.takeScreenshot(screenshotName);
```

Comment:
is iOS supported at this time? If not we should throw as well to make it obvious to the user

Reply:
iOS will come right after this PR.